src/auth/auth0-wrapper.test.js

```diff
@@ -0,0 +1,81 @@
+import { mount } from 'enzyme'
+import { default as createAuth0Client } from '@auth0/auth0-spa-js'
+import withAuth0 from './auth0-wrapper'
+
+jest.mock('@auth0/auth0-spa-js')
+jest.mock('axios')
+
+const Wrapped = () => <div />
+
+const originalError = console.error
+beforeAll(() => {
+  console.error = (...args) => {
+    if (/Warning.*not wrapped in act/.test(args[0])) {
```

Comment:
What is this about?

Reply:
This is a crummy workaround I found for testing `useEffect` with enzyme's `mount` method.  `useEffect` works the way you would expect if you use `mount`, but it logs a nasty warning.  This setup just swallows that error.  It's currently also in the `oauth-view.test.js`.  I think we can probably remove it once we upgrade enzyme.  I thought about trying to move it up into a more global setup, but then I forgot.  😁